.github/workflows/ci.yml

```
@@ -175,6 +190,8 @@ jobs:
         uses: astral-sh/setup-uv@v1
         with:
           enable-cache: true
+          version: ${{ env.UV_VERSION }}
+          version: ${{ env.UV_VERSION }}
```

YAMLlint (1.37.1)

[error] duplication of key "version" in mapping

(key-duplicates)